server/src/utils/validateMime.js

```diff
@@ -5,7 +5,7 @@ const Path = require('path')
 const allowedImageMimeTypes = ['image/jpeg', 'image/png']
 const imageMetadataSchema = Joi.object({
     headers: Joi.object({
-        'content-type': Joi.string().valid(...allowedImageMimeTypes),
```

Comment:
`.valid(...allowedImageMimeTypes)` had to be removed in otherwise the error returned by hapi would be the one from `Joi.assert` instead of `Boom.badRequest`.

Reply:
Was going to comment that we could add a message with `.message()`, but since we are validating it below either way, it is kind of superfluous anyway.